### ISSUE TITLE
[Components and pattern standards] Design components or patterns don't align with Design System guidelines. (04.07.4)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#55858]
#### VFS team name
VA Online Scheduling (VAOS)

#### VFS product name
VA Online Scheduling

#### VFS feature name
Appointment List

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Design components or patterns don't align with Design System guidelines.

### Issue Details
The Pending and Past navigation items are visually styled to be links and are navigational so they should be links, but they're coded as buttons. Buttons and links are treated differently by some assistive technologies, including screen readers and voice command software. When the code semantics of the element don't match up with the visual presentation or the interaction pattern for the element, you can introduce unexpected accessibility barriers. 

Example: a voice command user may speak the command "Click link," which will then highlight all the links on the page with a number, and then allow them to specify which link they want to follow, eg. "Click fifteen." Since Pending and Past are visually styled to look like links, a sighted voice command user would expect them to be highlighted when they say "Click link." But since they're not coded as links, they won't be highlighted --- leaving the user to try to guess what might be going wrong and what the magic words might be to be able to click those not-links.

### Link, screenshot or steps to recreate
The Pending and Past items are currently coded as:
```
<nav aria-label="Appointment list navigation" class="vaos-appts__breadcrumb">
   <ul>
      <li>
         <button id="pending" type="button" class="va-button-link">
            Pending (14)
         </button>
      </li>
      <li>
         <button id="past" type="button" class="va-button-link">
            Past
         </button>
      </li>
   </ul>
</nav>
```

### VA.gov Experience Standard
[Category Number 04, Issue Number 07](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

### Platform Recommendation
Maintain material honesty whenever possible. Since it looks like a link and acts like a link, it should just be a link.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by outerpress** [comment]
Just adding info to this ticket, @vbahinwillit did this work but we had to revert it because of issues Tony outlined in [54102](https://github.com/department-of-veterans-affairs/va.gov-team/issues/54102#issuecomment-1462437556).

We also saw Talkback skipping va-links ([slack thread](https://dsva.slack.com/archives/C8E985R32/p1678395616652319)) This may be related to va-link component not supporting aria-labels - #55857

**Comment by ldelacosta** [comment]
VAOS FE team completed the work [56168](https://app.zenhub.com/workspaces/vaos-team-603fdef281af6500110a1691/issues/gh/department-of-veterans-affairs/va.gov-team/56168)
